### ISSUE TITLE
custom payload not rendering in fulfillment message

### ISSUE THREAD

**Issue by Sarps** [eristemena/dialogflow-fulfillment-webhook-php#4]
When I create a custom payload programmatically and render it, it ends up being outside of the fulfillmentMessages array.

```php
$this->agent->reply(\Dialogflow\RichMessage\Payload::create([
            "type" => "web_url",
            "url" => "There are 24 shops in Lisda.",
            "title" => "http://localhost:8000/maps",
 ]));
```

```json
{"payload":{"unspecified":{"type":"web_url","url":"http:\/\/localhost:8000\/maps","title":"Click to view map."}},"fulfillmentMessages":[{"text":{"text":["There are 24 shops in Lisda."]}}],"fulfillmentText":"There are 24 shops in Lisda.","outputContexts":[]}
```

Due to that dialogflow places the payload in data object instead of messages object.

```json
{"speech":"There are 24 shops in Lisda.","source":"","messages":[{"type":0,"speech":"There are 24 shops in Lisda."}],"data":{"unspecified":{"type":"web_url","title":"Click to view map.","url":"http://localhost:8000/maps"}}}
```

Which is not what I get when I add the payload manually to the intent response inside dialogflow.

```json
"fulfillment":{"messages": [{"type":"custom_payload","platform":"google","payload":{"unspecified":{...}}},,{"type":4,"payload":{"unspecified":{"type":"web_url","url":"http://localhost:8000/maps","title":"Click to view map."}}}], "speech":""}
```

Please is there any way that I could force the payload into the fulfillmentMessages array.
Thank you for your good works. This package has really helped a lot.

**Comment by eristemena** [comment]
You're right, this is a bug. What is the [request source](https://github.com/eristemena/dialogflow-fulfillment-webhook-php/blob/master/docs/WebhookClient.md#webhookclientgetrequestsource) you're using?

**Comment by Sarps** [comment]
Thanks for the quick response. I am not worried about the key, I can handle that. My issue is that the payload is outside the *fulfillmentMessages* array.
Just like how the text response is inside *fulfillmentMessages*, I was hoping the payload would also be in there.
Thanks, once again.

**Comment by eristemena** [comment]
Oh sorry, are you talking about `payload` object inside `fulfillmentMessages` as per Dialogflow specification [here](https://dialogflow.com/docs/reference/api-v2/rest/Shared.Types/Message#SCHEMA_REPRESENTATION)?

**Comment by Sarps** [comment]
yh, exactly

**Comment by eristemena** [comment]
I can't reproduce the JSON response you mentioned in the first comment, either by adding it manually using DIalogflow console or using Dialogfloww official NodeJS library. 

Can you explain more what is the use case here? And what is `web_url` type? is it a Facebook messaging template? if so, why does your JSON contain `google` in the `platform`?

**Comment by basiteaal** [comment]
I think this issue should be closed due to inactivity.
